### ISSUE TITLE
Add group infos from backend to group profiles

### ISSUE THREAD

**Issue by sebgrebe** [rtcharity/hubreboot#79]

**Comment by sebgrebe** [comment]
Implements [this wireframe](https://github.com/rtcharity/hubreboot/issues/60)

Some examples:
* [Group with summary; country in key facts](https://eahub.azurewebsites.net/group/96) 
* [Group with summary; organiser and website in key facts](https://eahub.azurewebsites.net/group/97)
* [Group with summary; donations; no key facts](https://eahub.azurewebsites.net/group/116) 
* [Group with more than one organiser in key facts](https://eahub.azurewebsites.net/group/94) 

Remarks
* A couple of fields in the key facts that **were** in the wireframes I didn't implemented as they don't exist in the database
* I've implemented a few fields in the key facts that were **not** in the wireframes, which I implemented as they were in the database and seemed useful 

@Richenda Leave feedback below.

**Comment by Richenda** [comment]
https://github.com/rtcharity/hubreboot/issues/86 @sebgrebe made this post to summarise things we should add that weren't in the database and things in the original database that we can collapse or remove.

For this example https://eahub.azurewebsites.net/group/94: We should not display any personal email addresses for privacy reasons. Only group email addresses. I can help with this on a case by case basis. Is it possible to send me a csv or google sheet of all the data from the Hub for groups and I can manually go through each one and update using current LEAN records. I know it's labour intensive but that's kind of how LEAN data has always worked because group organisers can't be relied on to keep things updated (too much changeover of committees) and we have to constantly do detective work.
To summarise, what we should see next to 'Group Organiser' is just a name. And if that person has a profile on the Hub then the name is also a link to their profile.

https://eahub.azurewebsites.net/group/116 Loving the section for group donations. Looks great.

https://eahub.azurewebsites.net/group/97 The default link colour around the site was the darker green/teal, right? Can we try have links white and underlined but teal green when you click them or hover on them? How it looks right now is too busy I think.

https://eahub.azurewebsites.net/group/96 Great, looking good! For the button next to the group name, can we show 'join group' for ordinary users. And for the member that is the organiser of this group, they should see 'edit group' here.

**Comment by sebgrebe** [comment]
@Richenda Thanks, I've implemented this all now (you might not see this on the azure page yet, we have to deploy first)
